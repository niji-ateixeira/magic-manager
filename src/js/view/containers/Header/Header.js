import React from 'react';
import PropTypes from 'prop-types';
import injectSheet from 'react-jss';

import styles from './HeaderStyles';

const Header = ({ classes }) => (
  <header className={classes.Container}>
    <span className={classes.Text}>
      Bienvenue sur Magic Manager
    </span>
  </header>
);

Header.propTypes = {
  classes: PropTypes.shape({}).isRequired,
};

export default injectSheet(styles)(Header);
