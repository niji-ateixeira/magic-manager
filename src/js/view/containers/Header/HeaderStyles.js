const styles = {
  Container: {
    background: 'black',
    padding: 20,
    flex: 1,
    flexDirection: 'collumn',
    justifyContent: 'center',
    alignItems: 'center',
  },
  Text: {
    color: 'white',
  },
};

export default styles;
