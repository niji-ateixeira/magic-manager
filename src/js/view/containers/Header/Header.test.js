import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header.js';

test('renders without crashing', () => {
  const component = renderer.create(
    <Header />,
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
