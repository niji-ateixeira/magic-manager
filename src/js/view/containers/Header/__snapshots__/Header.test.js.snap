// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<header
  className="Header-Container-0-1-1"
>
  <span
    className="Header-Text-0-1-2"
  >
    Bienvenue sur Magic Manager
  </span>
</header>
`;
